projects/dsp-ui/src/lib/viewer/views/list-view/resource-grid/resource-grid.component.html

```diff
@@ -2,12 +2,12 @@
     <mat-card
         class="grid-card link"
         [class.selected-resource]="selectedResourceIdx === i"
-        *ngFor="let res of resources.resources; let i = index;"
-        (click)="resourceSelected.emit(res.id)">
+        *ngFor="let res of resources.resources; let i = index;">
         <!-- TODO: add the representation preview here, mat-card-image can be used for images -->
         <mat-card-header class="grid-card-header">
             <mat-card-subtitle class="res-class-label">{{ res.entityInfo.classes[res.type].label }}</mat-card-subtitle>
             <mat-card-title class="res-class-value">{{ res.label }}</mat-card-title>
+            <mat-checkbox id="{{ i }}" (change)="viewResourse($event.checked, res.id)" class="res-checkbox"></mat-checkbox>
```

Comment:
It would be better to have the checkbox in the top right corner (header action area). But at the moment, the grid view is not used in DSP-APP. Maybe we will redesign card later